api/src/pcapi/routes/serialization/public_api_collective_offers_serialize.py

```diff
@@ -115,6 +115,19 @@ def validate_price_detail(price_detail: str | None) -> str | None:
     return price_detail
 
 
+def validate_students(students: list[str] | None) -> list[StudentLevels]:
+    if students is None or len(students) == 0:
+        raise ValueError("La liste des niveau scolaire ne peux pas être vide")
```

Comment:
style + typos

```suggestion
    if not students:
        raise ValueError("La liste des niveaux scolaire ne peut pas être vide")
```

(De plus, le message d'erreur paraîtrait moins technique avec "Veuillez sélectionner au moins un niveau scolaire." Mais je n'ai pas le contexte, donc ma proposition n'est peut-être pas pertinente.)

Reply:
On est dans le cas d'une API publique donc on s'adresse a des devs (et sinon je prend la proposition de changement du code)